fapolicy_analyzer/setup_cmds.py

```diff
@@ -0,0 +1,208 @@
+# Copyright Concurrent Technologies Corporation 2022
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+
+import os
+import shutil
+from distutils import dir_util
+from distutils.cmd import Command
+from glob import glob
+from os import path
+from pathlib import Path
+
+
+class parse_help(Command):
+    description = "Update source controlled help documentation files from online source"
+    user_options = [
```

Comment:
We will need some way to filter wiki paths... including/excluding individual paths may become tedious and error prone.

I have a few thoughts, nothing that seems foolproof.

The easiest one is adding a hash tag identifier to the pages that need included.

Eg.
```
My Wiki Page
===

- one thing
- two thing
- three thing

#user-guide
```

Reply:
To simplify this we are going to do a single page user doc on the wiki.  Future use may require a more complicated layout, but this should suffice for the foreseeable future.